### PR TITLE
Fix text selection in dark mode and modal view

### PR DESCRIPTION
**Pull request by juliushaertl**
* Resolves: #3711
* Target version: master 

### Summary

- Ensure that text in the modal is selectable (as it is blocked by the modal components touch handling)
- Fix selection background color in dark mode

### CODE REVIEW

File: src/components/card/CardSidebar.vue [hunk 1]
```diff
@@ -225,6 +225,9 @@ export default {
 		max-width: calc(100% - #{$modal-padding*2});
 		padding: 0 14px;
 		max-height: 100%;
+		user-select: text;
+		-webkit-user-select: text;
```
Comment (luka-nextcloud) [inline src/components/card/CardSidebar.vue:229]: What do you think about the support for other browsers?

-moz-user-select: text;
-webkit-user-select: text; 
-ms-user-select: text; 
user-select: text;

Comment (juliushaertl) [reply src/components/card/CardSidebar.vue:229]: All recent major browsers should be fine without the prefix except for safari, according to https://caniuse.com/mdn-css_properties_user-select ;)

**Review by luka-nextcloud — APPROVED** [review]

**Pull request merged**